dist/stampit.js

```diff
@@ -3786,20 +3786,25 @@ function addMethods(fixed, methods) {
   mixer.mixInFunctions.apply(null, args);
   return fixed.methods;
 }
-function addState(fixed, states) {
-  var args = [fixed.state].concat(states);
-  fixed.state = mixer.merge.apply(null, args);
-  return fixed.state;
+function addRefs(fixed, states) {
+  var args = [fixed.refs].concat(states);
+  fixed.refs = fixed.state = mixer.mixIn.apply(null, args);
+  return fixed.refs;
 }
 function addEnclose(fixed, encloses) {
   encloses = isArray(encloses) ? extractFunctions.apply(null, encloses) : extractFunctions(encloses);
   fixed.enclose = fixed.enclose.concat(encloses);
   return fixed.enclose;
 }
+function addProps(fixed, propses) {
+  var args = [fixed.props].concat(propses);
+  fixed.props = mixer.merge.apply(null, args);
+  return fixed.props;
+}
 
 function cloneAndExtend(fixed, extensionFunction, args) {
   args = arguments.length > 3 ? slice(arguments, 2, arguments.length) : args;
-  var stamp = stampit(fixed.methods, fixed.state, fixed.enclose);
+  var stamp = stampit(fixed.methods, fixed.refs, fixed.enclose, fixed.props);
```

Comment:
Let me add a unit test on that. It's better to be as backwards compatible as possible.